### PR TITLE
Updated Achievements and Gallery

### PR DESCRIPTION
**Pull request by samarthbc**
Made a card to iterate over achievements to display all the achievements and updated gallery page to include pics event wise

### CODE REVIEW

File: src/app/achievements/page.tsx
Comment (Aswinr24) [inline src/app/achievements/page.tsx:1]: I guess this is the achievements object:
```
{

	id: number,
	created_at: time,
	field: string,
	description: string,
	image: string,
	achievement: string,
	achievement_date: string,
}
```
just change this accordingly and I think tags are not needed here
also can you revamp the achievement cards design a bit

File: src/app/gallery/page.tsx
Comment (Aswinr24) [inline src/app/gallery/page.tsx:1]: Gallery LGTM
if you have time then do try to enchance the design

File: src/app/gallery/page.tsx [hunk 1]
```diff
@@ -1,7 +1,38 @@
 import React from "react";
```
Comment (Aswinr24) [inline src/app/gallery/page.tsx:1]: Gallery LGTM
if you have time then try to enhance its design

**Review by Aswinr24 — CHANGES_REQUESTED** [review]
Code Lgtm, but try to get the design of the achievements cards something like the team members cards in this, I mean similar design scheme @samarthbc 

https://deploy-preview-14--aegisdsce.netlify.app/teams

**Review by sahil-raj — APPROVED** [review]

**Review by Aswinr24 — APPROVED** [review]

**Pull request merged**